frontend/src/assets/css/typography-variables.css

```diff
@@ -0,0 +1,74 @@
+:root {
+    --font-weight-400: 400;
+    --font-weight-500: 500;
+    --font-family: 'Rubik', sans-serif;
+
+    /* Heading 1 */
+    --font-size-level-1-heading: 44px;
+    --line-height-level-1-heading: 64px;
+
+    /* Heading 2 */
+    --font-size-level-2-heading: 30px;
+    --line-height-level-2-heading: 48px;
+
+    /* Heading 3 */
+    --font-size-level-3-heading: 20px;
+    --line-height-level-3-heading: 32px;
+
+    /* Heading 4 */
+    --font-size-level-4-heading: 18px;
+    --line-height-level-4-heading: 24px;
+
+    /* Heading 5 */
+    --font-size-level-5-heading: 16px;
+    --line-height-level-5-heading: 18.5px;
+
+    /* Heading 6 */
+    --font-size-level-6-heading: 14px;
+    --letter-spacing-level-6-heading: 0.75px;
+    --line-height-level-6-heading: 24px;
+
+    /* Subheading 2 */
+    --font-size-sub-heading: 24px;
+    --line-height-sub-heading: 48px;
+
+    /* Body 1 */
+    --font-size-level-1-body: 16px;
+    --line-height-level-1-body: 27.5px;
+
+    /* Body 2 */
+    --font-size-level-2-body: 14px;
+    --line-height-level-2-body: 19.5px;
+    --line-height-level-2-body-semi-bold: 19px;
+
+    /* Caption */
+    --font-size-caption: 12px;
+    --line-height-caption: 19.5px;
+
+    /* Quotation */
+    --font-size-quotation: 16px;
+    --font-style-quotation: italic;
+    --line-height-quotation: 24px;
+
+    /* Input label */
+    --font-size-input-label: 16px;
+    --font-weight-input-label: 400;
+    --line-height-input-label: 16px;
+
+    /* Input text */
+    --font-size-input: 15px;
+    --letter-spacing-input: -1%;
```

Comment:
I think it unnecessary

Reply:
Done in commit [c829865](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/24/commits/c82986598eeb62c53f77120da7f753c700286dd9)